src/css/forms.scss

```diff
@@ -285,7 +285,7 @@ p.sentence-case:first-letter {
 
 .radio__title {
     display: inline-block;
-    font-size: 12px;
+    font-size: 13px;
```

Comment:
Is this intentional? as this will be reflected in all sections where this class is used.

Reply:
Yes